specs/beacon-chain.md

```diff
@@ -571,6 +572,7 @@ def add_validator(validators: List[ValidatorRecord],
                   withdrawal_shard: int,
                   withdrawal_address: Address,
                   randao_commitment: Hash32,
+                  status: uint8,
```

Comment:
In Python, we didn't define `uint8`, `uint16`... .etc for type hints. At runtime, it's just `int`, and the types of fields will be checked during data serialization.